libsrc/telestrat/kbhit.s

```diff
@@ -10,12 +10,10 @@
 
 _kbhit:
         BRK_TELEMON XRD0
-        bcs     @no_char_action
         ldx     #$00 
+        bcs     @no_char_action
         lda     #$01
         rts
 @no_char_action:
-        lda     #$00
         tax        
```

Comment:
This needs to be "txa"

Reply:
Yes good point. I will fix it tonight. Thanks for the point :)

Reply:
> This needs to be "txa"

I don't agree.

Reply:
why not?

Reply:
Just to make sure we're talking about the same code. This ias what you want to see? `tax` replaced with `txa`, right?

```
_kbhit:
        BRK_TELEMON XRD0
        ldx     #$00 
        bcs     @no_char_action
        lda     #$01
        rts
@no_char_action:
        lda     #$00
        txa        
        rts
```

My take is that the `tax` shouldn't be replaced with `txa`. Rather it should be removed altogether.

Reply:
I was thinking about something like:

```
_kbhit:
        BRK_TELEMON XRD0
        ldx     #$00 
        bcs     @no_char_action
        lda     #$01  ; return one
        rts
@no_char_action:
        txa        ; return zero
        rts
```

Reply:
But that's not what you wrote above ;-)